packages/x-charts/src/ChartsSurface/ChartsSurface.tsx

```diff
@@ -45,9 +45,16 @@ const ChartsSurfaceStyles = styled('svg', {
   touchAction: 'pan-y',
   userSelect: 'none',
   gridArea: 'chart',
-  '&[data-has-focused-item=true]': {
-    // Move the focus outline responsibility to children
-    outline: 'none',
+  '&:focus': {
+    outline: 'none', // By default don't show focus on the SVG container
+  },
+  '&:focus-visible': {
+    // Show focus outline on the SVG container only when using keyboard navigation
+    outline: 'auto',
+    '&[data-has-focused-item=true]': {
+      // But not if the chart has a focused children item
+      outline: 'none',
+    },
```

Comment:
@JCQuintas I went further with the focus display.

By relying on the `:focus-visible` the outlined is not display when the element get focused by click, but is visible when focused by <kbd>Tab</kbd>

Reply:
Nice, interactions feels good now, thanks :D